docs/concepts/policy/pod-security-policy.md

```diff
@@ -417,6 +418,20 @@ containers, and abusing the credentials of system services, such as Kubelet._
 **ReadOnlyRootFilesystem** - Requires that containers must run with a read-only
 root filesystem (i.e. no writeable layer).
 
+### FlexVolume drivers
+
+When the [`Volumes`](#volumes-and-file-systems) field contains `flexVolume` in
+its list value, the cluster admin can further specify which driver(s) is permitted
+by setting the `allowedFlexVolumes` field.
+
+**AllowedFlexVolumes** - Provides a whitelist of allowed FlexVolumes. Empty or
+nil indicates that all FlexVolume drivers may be used. For example, the following
+setting only permits the `examle/fast_cache` driver to be used on nodes:
+
+```yaml
+allowedFlexVolumes: [ "example/fast_cache" ]
```

Comment:
Unfortunately, this example incorrect and won't work :-|

`allowedFlexVolumes` accepts list of the objects (not list of the strings). See correct example in the original PR from @wanghaoran1988

Reply:
My fault. It should be a list of objects where each object has a driver property. Will fix.